http-tests/src/test/scala/org/apache/pekko/http/scaladsl/server/ConnectionTestApp.scala

```diff
@@ -43,10 +43,11 @@ object ConnectionTestApp {
 
   val clientFlow = Http().superPool[Int]()
 
-  val sourceActor = {
+  val sourceQueue = {
     // Our superPool expects (HttpRequest, Int) as input
     val source =
-      Source.actorRef[(HttpRequest, Int)](10000, OverflowStrategy.dropNew).buffer(20000, OverflowStrategy.fail)
+      Source.queue[(HttpRequest, Int)](10000)
+        .buffer(20000, OverflowStrategy.fail)
```

Comment:
Is this a change in behavior (dropNew -> fail)? Seems fine for a test app though.

Reply:
> Would https://pekko.apache.org/japi/pekko/1.2/org/apache/pekko/stream/OverflowStrategy.html#backpressure() make sense? OverflowStrategy.dropNew is being removed (already deprecated in 1.x).

TBH I'm not sure what this means in practice here - would that mean 'offer' would give a return value indicating the backpressure? I think `fail` is probably a fine choice.